### PR TITLE
1letter/collection behavior integration

### PR DESCRIPTION
**Pull request by 1letter**
add integration for collection behavior

### CODE REVIEW

File: src/collective/collectionfilter/configure.zcml [hunk 1]
```diff
@@ -1,79 +1,86 @@
 <configure
```
Comment (thet) [inline src/collective/collectionfilter/configure.zcml:1]: Hi, I'm -1 on the reformatting.
The Plone and Zope style guides say:
```
Indentation of 2 characters to show nesting, 4 characters to list attributes on separate lines.
This distinction makes it easier to see the difference between attributes and nested elements.
```

https://docs.plone.org/develop/styleguide/python.html#indentation
https://zopetoolkit.readthedocs.io/en/latest/codingstyle/zcml-style.html

File: src/collective/collectionfilter/portlets/__init__.py [hunk 2]
```diff
@@ -27,4 +30,9 @@ def reload_url(self):
 
     @property
     def available(self):
+        if not (
```
Comment (thet) [inline src/collective/collectionfilter/portlets/__init__.py:33]: Please add a comment why you are disabling the availability for those interfaces here.

**Review by thet — CHANGES_REQUESTED** [review]
Thanks for the PR!
I'm probably +1 if I would knew what this PR is about. I have no ides what it does.
Please add a CHANGELOG entry where you explain the reasoning about this.
You can also add the same changelog entry for the commit message - that is what I most often do.
Also, please note my other two comments in the code. Basically, I'm -1 on the ZCML format changes. Is that an automatically formatted change? If yes, which tool are you using?
In the Plone/Zope world I recommend to use https://github.com/collective/zpretty

File: src/collective/collectionfilter/portlets/__init__.py (hunk 2 above)
Comment (1letter) [reply src/collective/collectionfilter/portlets/__init__.py:33]: @thet 

Thanks for comment!

Use Case:

1. enable Collection Behavior for Folders
2. add a Folder
3. add some Documents to the Folder with different Subjects
4. configure the Collection, e.g. show only Documents
5. assign a Collection Filter Portlet to the Folder, use "Subject" as Criteria, point to the Folder itself
6. Filter the Collection via Portlets and Click on a Result Item

Now the Portlets are empty or it make no sense, that the Filter Options available on the Children Detail View. That is the reason why i restricted the rendering to the Interfaces "ISyndicatableCollection" and "ICollectionish". It means, only if the Context a Collection, show the Portlets. Maybe this to strictly ?

File: src/collective/collectionfilter/configure.zcml (hunk 1 above)
Comment (1letter) [reply src/collective/collectionfilter/configure.zcml:1]: @thet 
Thanks for the hint, good to know, i use vscode with "autoformat on save option" and the "XML Language Support by Red Hat" Plugin. I will refactor the zcml files with zpretty.

**Comment by 1letter** [comment]
I close this PR. No changes needed for Use Case.

1 . I register  an adapter in my project to use this add-on in a folder with enabled collection-behavior.
2. The Folder is restricted to add only one Contenttpye (Collection as Contenttype is excluded)
3.  I register my own Portletmanager to use the collective.collectionfilter Portlets only in the context of the folder, not in the context of the filterable childs in folder.

**Pull request closed without merging**